actions/src/sema4ai/actions/_commands.py

```diff
@@ -536,6 +579,14 @@ def run(
                     return retcode
                 else:
                     retcode = int(any(action.failed for action in actions))
+                    # retcode = 0
+                    # for action in actions:
+                    #     if action.failed:
+                    #         if action.result is None:
+                    #             # This is the case where an ActionError is raised
+                    #             # (so, we have an exception but also a proper result).
+                    #             retcode = 1
+                    #             break
```

Comment:
Does it help to have dead code here? (or is this more like a reference of a potential upcoming TODO)